src/python/geoclaw/surge/storm.py

```diff
@@ -231,5 +488,32 @@ def available_models():
 
 
 if __name__ == '__main__':
+    
```

Comment:
Due to some weirdness in `nosetests` all tests and data are kept in the `$CLAW/geoclaw/tests` subdirectory.  There should be a couple of test modules in there already and these should be added along with the test data to the appropriate directory.